### DISCUSSION TITLE
Why every `dup` syscall doesn't call eBPF function `sys_exit_dup_tail()`

### DISCUSSION THREAD

**Discussion by liusy58** [aquasecurity/tracee#1899] [Questions and Help] [answered]
Hi, sir, I build `tracee-ebp` succefully and run it, and I write a program to call `dup` syscall, and I add a log using `bpf_print` in `sys_dup_exit_tail` . But I got nothing when I execute `sudo cat /sys/kernel/debug/tracing/trace_pipe`. Any help?

**Comment by AsafEitani** [comment, marked as answer]
Hi!
Thanks for letting us know!
I have opened a PR for a fix - please check if that fixes that for you :)
#1905
